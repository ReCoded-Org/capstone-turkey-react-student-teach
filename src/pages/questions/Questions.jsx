import Question from '../../components/Question/Question';

function Questions() {
  return <Question />;
}

export default Questions;
