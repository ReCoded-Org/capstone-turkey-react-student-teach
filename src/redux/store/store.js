import { configureStore } from '@reduxjs/toolkit';
import addQuestionSlice from '../features/addQuestionSlice';

const store = configureStore({
  reducer: {
    addQuestionReducer: addQuestionSlice,
    signIn: userSlice,
  },
});

export default store;
