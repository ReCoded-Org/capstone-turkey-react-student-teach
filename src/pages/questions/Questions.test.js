import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Questions from './Questions';
import store from '../../redux/store/store';

it('Navbar snapshot', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Questions />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
