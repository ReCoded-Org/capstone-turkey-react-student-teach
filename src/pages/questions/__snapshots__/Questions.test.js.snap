// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot 1`] = `
<div
  className="bg-zinc-100  pt-5 pb-[5rem]"
>
  <div
    className=" container mx-auto w-full pt-8 rounded-md transition-all ease-in-out duration-300  text-center lg:text-left bg-white text-black "
  >
    <section
      className="flex min-h-[9vh] lg:min-h-[15vh]"
    >
      <div
        className="relative  hidden min-w-[70vw] lg:flex flex-col items-center"
      >
        <div
          className="group flex absolute top-[50%] left-[50%] -translate-y-2/4 -translate-x-2/4 bg-[#CA7560] rounded-[40px] p-2.5 "
        >
          <input
            className="group-hover:w-52  group-hover:py-2.5 w-0 border-none outline-none focus:ring-transparent p-0 bg-transparent border-0 text-[1.1rem] duration-500 ease-in text-white "
            onChange={[Function]}
            placeholder="search...."
            style={
              Object {
                "twRingColor": "none",
              }
            }
            type="text"
          />
          <span
            className="group-hover:bg-[#CA7560] hover:rotate-[360deg] scale-75 text-black float-right w-10 text-[1.3rem] h-10 rounded-full bg-[#CA7560] flex justify-center duration-500 cursor-pointer no-underline "
            type="button"
          >
            <img
              alt=""
              src="search.svg"
            />
          </span>
        </div>
      </div>
    </section>
    <div
      className="mt-5 md:mr-7 lg:mr-10 pb-10"
    >
      <div
        className="flex flex-col items-center my-12"
      >
        <div
          className="flex text-gray-700"
        >
          <div
            className="h-12 w-12 mr-1 flex justify-center items-center rounded-full bg-gray-100 cursor-pointer"
            onClick={[Function]}
            type="button"
          >
            <svg
              className="feather feather-chevron-left w-6 h-6"
              fill="none"
              height="100%"
              stroke="currentColor"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              viewBox="0 0 24 24"
              width="100%"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="15 18 9 12 15 6"
              />
            </svg>
          </div>
          <div
            className="flex h-12 font-medium rounded-full bg-gray-100"
          >
            <div
              className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full bg-[#CA7560] text-white  "
              onClick={[Function]}
              type="button"
            >
              1
            </div>
            <div
              className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full    "
              onClick={[Function]}
              type="button"
            >
              2
            </div>
            <div
              className="w-12 md:flex justify-center items-center hidden  cursor-pointer leading-5 transition duration-150 ease-in  rounded-full  "
              onClick={[Function]}
              type="button"
            >
              3
            </div>
          </div>
          <div
            className="h-12 w-12 ml-1 flex justify-center items-center rounded-full bg-gray-100 cursor-pointer "
            onClick={[Function]}
            type="button"
          >
            <svg
              className="feather feather-chevron-right w-6 h-6"
              fill="none"
              height="100%"
              stroke="currentColor"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              viewBox="0 0 24 24"
              width="100%"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="9 18 15 12 9 6"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
    <div />
  </div>
</div>
`;
